README.md

```diff
@@ -5,7 +5,7 @@
 This repository exists for developing [PEP 484](https://www.python.org/dev/peps/pep-0484/)
```

Comment:
Could you also kindly update `numpy_stubs` to `numpy-stubs` on the first line of the readme?